lemonparty2018.py

```diff
@@ -30,10 +35,25 @@ def static_path_processor():
     }
 
 
+def login_required(f):
+    @wraps(f)
+
+    def decorated_function(*args, **kwargs):
+        if not DEBUG and not session.get('is_authenticated'):
+            return redirect('/')
+
+        return f(*args, **kwargs)
+
+    return decorated_function
+
+
+# pages
+# ------------------------------------------------------------------------------
+
 @app.route('/')
 def index():
     if session.get('is_authenticated'):
-        return render_template('home.html')
+        return redirect('/home')
```

Comment:
Maybe we can try named routes. Perhaps `return redirect(url_for('home')` would work?